projects/convex-hull/index.md

```diff
@@ -42,8 +42,16 @@ problem and the different algorithms to solve it.
 
 ## Testing
 
-Verify that the actual output matches the expected output. See the
-[requirements][3] section for an example of the expected output.
+The following table contains various test cases that you can use to
+verify the correctness of your solution; please note that different algorithms
+could produce the output starting from a different point, and/or in the opposite direction:
+
+| Description         | Input X | Input Y | Output |
+|---------------------|---------|---------|--------|
+| X-Ordered triangle  | 100, 180, 240                         | 220, 120, 20                         | (100, 220)<br>(240, 20)<br>(180, 120)                            |
```

Comment:
```suggestion
| X-Ordered triangle  | 100, 180, 240                         | 220, 120, 20                         | 100, 220<br>240, 20<br>180, 120                            |
```

I've been thinking about this output and I'm wondering if it should mirror the input. In other words, a list of x values and a list of y values. Also, to simplify formatting, I think we should try to avoid the parentheses if possible.

That said, I'm not sure what the best move is here. Perhaps what you have is fine since the minimum number of points would be 3, so there's no way to confuse the output format with the input format.

Reply:
If you agree with my proposal above, I believe having the output as (x, y) pairs is easier to test

Reply:
Testing isn't so much an issue as having other languages implement similar solutions. I find that text formatting tends to get in the way of just implementing the algorithms—which is why I proposed removing the parentheses. I don't think any of our other algorithms retain the parentheses in the output (e.g. [sorting](https://sample-programs.therenegadecoder.com/projects/bubble-sort/). That said, I'll let @auroq weigh in since they might have other thoughts. I don't have any opinion either way.